src/main/java/com/epam/esm/utils/InitialOrderDataHolder.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.utils;
+
+public class InitialOrderDataHolder {
```

Comment:
Why such name? Why not `OrderDto`? Or `OrderRequestDto`?

Reply:
Fixed